w3-blob.md

```diff
@@ -325,11 +351,76 @@ It is RECOMMENDED that issued `BlobAddress` only accept `PUT` payload that match
 
 Allocation MUST have no effects.
 
+## Put Blob
+
+Any agent MAY perform `/http/put` capability invocation on behalf of the subject. [Add blob] capability provider MUST add `/http/put` effect and capture private key of the `subject` in the `meta` field so that any agent could perform it.
+
+An agent that invoked [add blob] capability is expected to perform this task and issue receipt on completion.
+
+### Put Blob Capability
+
+#### Put Blob Capability Schema
+
+> ℹ️ In the UCAN 0.9 `meta` is unknown as `fct` field
+
+```ts
+type BlobPut = {
+  cmd: "/http/put"
+  sub: DID
+  args: {
+    address: BlobAddress
```

Comment:
Looks like content is missing here?

Reply:
or at least, this is not in sync with above. I would like to see content here anyway, as we should be able to query for the content rather than needing to know how a write target encodes it in the address (which actually may even not happen)

Reply:
```suggestion
    content: Multihash
    address: BlobAddress
```